ocean_lib/ocean/ocean_compute.py

```diff
@@ -192,6 +192,7 @@ def start(
         algorithm_data_token: str = None,
         output: dict = None,
         job_id: str = None,
+        raw_response: bool = False,
```

Comment:
same as above, don't need additional parameter, just return the raw response always

Reply:
but this would be a breaking change

Reply:
That's why I didn't want to cause another breaking change and I add an additional parameter instead.